SimPPS/RPDigiProducer/python/RPSiDetConf_cfi.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+
+RPSiDetDigitizer = cms.EDProducer("RPDigiProducer",
```

Comment:
It would be better to implement `fillDescriptions()` in `RPDigiProducer` and remove this file.